workers/unity/Assets/Playground/Scripts/MonoBehaviours/UpdateSpinnerColor.cs

```diff
@@ -4,22 +4,29 @@
 using UnityEngine;
 using Color = Generated.Playground.Color;
 
+#region Diagnostic control
+
+// Disable the "variable is never assigned" for injected fields.
+#pragma warning disable 649
+
+#endregion
+
 namespace Playground.MonoBehaviours
 {
     public class UpdateSpinnerColor : MonoBehaviour
     {
-        [Require] private SpinnerColor.Requirable.Writer writer;
+        [Require] private SpinnerColor.Requirable.Writer spinnerColorWriter;
```

Comment:
I'm not sure I agree that we need to have longer names in unambiguous contexts. Although I don't really mind one way or the other.

Reply:
I think we need to set a good example in our code, and `writer` isn't doing that, IMO.